WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -394,6 +407,25 @@ extension OrderListViewController {
     private func ensureFooterSpinnerIsStopped() {
         footerSpinnerView.stopAnimating()
     }
+
+    /// Attempt showing details for first item if:
+    /// - the order detail screen is not already displayed
+    /// - the split view is not collapsed
+    /// - there's available data
+    ///
+    private func attemptShowingDetailsForFirstItem() {
+        let secondaryColumnNavigationController = splitViewController?.viewController(for: .secondary) as? UINavigationController
+        let displayedOrderDetail = secondaryColumnNavigationController?.viewControllers.first as? OrderDetailsViewController
+
+        guard displayedOrderDetail == nil,
+              !dataSource.isEmpty,
+              splitViewController?.isCollapsed == false else {
+            return
+        }
+        let firstIndexPath = IndexPath(row: 0, section: 0)
+        tableView.selectRow(at: firstIndexPath, animated: false, scrollPosition: .top)
```

Comment:
nit: maybe this is superfluous since we are already calling below `tableView(tableView, didSelectRowAt: firstIndexPath)`?

Reply:
The `selectRow` method only highlight the specified row and doesn't trigger the delegate method, therefore I need both methods to both highlight the row and trigger the selection.